lib/cookie_store.ex

```diff
@@ -0,0 +1,17 @@
+defmodule CookieStore do
+  import Plug.Conn
+
+  def read_arabic_value(conn) do
```

Comment:
This function doesn't seem to be related to cookies at all. Not sure it fits in this module.